cmd/contour/contour.go

```diff
@@ -123,6 +116,11 @@ func main() {
 		// buffer notifications to t to ensure they are handled sequentially.
 		buf := k8s.NewBuffer(&g, t, log, 128)
 
+		// this is necessary due to #113 wherein glog neccessitates a call to flag.Parse
```

Comment:
As much as I am a fan of this salty comment (when #118 was fixed I explicitly asked for this comment because I was so pissed off) we should probably change the comment to something more helpful for future archeologists.

(also the issue number in the comment is wrong, it should be 118, not 113 -- oops)

```
// client-go uses glog which requires initialisation as a side effect of calling
// flag.Parse (see #118 and https://github.com/golang/glog/blob/master/glog.go#L679)
// However, kingpin owns our flag parsing, so we defer calling flag.Parse until 
// this point to avoid the Go flag package from rejecting flags which are defined
// in kingpin. See #371